src/graph/binding.js

```diff
@@ -0,0 +1,29 @@
+import { camelCase } from 'lodash';
```

Comment:
Another thing to note: all our other js code uses 4 spaces but this repo uses 2. Will standardize on 4 in a follow up ticket.

Reply:
I'm wondering if this belongs in `nodule-config` or should we put it in `nodule-graph` to keep things segregated like in the good old days 😆

Reply:
I'm wondering if this belongs in `nodule-config` or should we put it in a new `nodule-graph`

Reply:
In practice, the idea of an object graph constructed from user-generated factories is a) in part what `bottle.js` does and b) very much dependent on config-loading.

If you look at `microcosm`, both abstractions (`object_graph` and config `loaders`) live in the same place because we need the latter to populate the former. That said, configuration loading ought to be configurable and extensible. (In the `microcosm` world, DynamoDB loading lives in another repo.)